lambdas/inventory/handler.py

```diff
@@ -76,8 +76,7 @@ def handler(event, context):
             if metadata_filename and metadata_type:
                 file_obj["assets"] = {metadata_type: metadata_filename}
             for key, value in event.items():
-                if "asset" in key:
-                    file_obj[key] = value
+                file_obj[key] = value  # Pass ALL keys along
```

Comment:
What is it that lead you to do this ? In other words, which keys wasn't passed along that you needed?

Noting again here how messy the way we handle parameters across steps, which we mentioned in https://github.com/NASA-IMPACT/active-maap-sprint/issues/637.

Reply:
It was probably just an oversight, but we should've just been passing everything along to begin with. start_datetime and end_datetime were also not being passed - for SRTMG1 there's no date in the filename so it kept erroring out. An alternative would be to do a collection lookup and grab the start/end dates but this seemed the simpler of the fixes

Reply:
Got it thanks !